### DISCUSSION TITLE
Gap when going to the previous page in one line (dmenu-like) mode/theme

### DISCUSSION THREAD

**Discussion by lominoss-git** [davatorium/rofi#1661] [Q&A] [answered]
![rofi](https://user-images.githubusercontent.com/79030093/180892710-1676571e-ac2d-4bfe-9c51-ee3050712490.gif)

As you can see, there is no gap on the left of the listview when I go to the next page, but when I go to the previous page a wide gap appears for some reason.

Note: This is a slightly modified version of the config found in [your official config file](https://github.com/davatorium/rofi-themes/blob/master/Official%20Themes/dmenu.rasi).

Here's my [config](https://github.com/lominoss-git/awesome-dots/blob/main/rofi/config.rasi).

**Comment by DaveDavenport** [comment, marked as answer]
? what is the question.

**Reply by lominoss-git** [reply]
Oh I'm sorry, I'm looking for a way to get rid of it. Thanks in advance

**Reply by DaveDavenport** [reply]
The gap moves from left/right depending on scroll direction. This 'mode' is different as it shows a dynamic number of elements, so I don't know the width/number of elements before I start iterating over the components, because I don't want to iterate twice (first to get width, 2nd to actually render it.) it works this way.
I don't use this mode and never had the need to change this.

If you want fixed element width, use a normal listview with lines: 1 and columns: X.  you will get fixed with elements that fill up the whole width (without a gap, if there are enough elements to display).

**Reply by lominoss-git** [reply]
Thank you so much.
